enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/database/OnboardingOtpRepository.java

```diff
@@ -44,7 +44,8 @@ public interface OnboardingOtpRepository extends CrudRepository<OnboardingOtpEnt
     @Query("UPDATE OnboardingOtpEntity o SET " +
             "o.status = com.wultra.app.enrollmentserver.model.enumeration.OtpStatus.FAILED, " +
             "o.timestampLastUpdated = CURRENT_TIMESTAMP, " +
-            "o.errorDetail = '" + OnboardingOtpEntity.ERROR_EXPIRED + "' " +
+            "o.errorDetail = '" + OnboardingOtpEntity.ERROR_EXPIRED + "', " +
```

Comment:
Shiouldn't we update `timestampFailed` here when the OTP expired?

Reply:
Yes, we should.